Nodejs/Product/Nodejs/Snippets/1033/Nodejs/class.snippet

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets
+    xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
+  <CodeSnippet Format="1.0.0">
+    <Header>
+      <Title>class</Title>
+      <Description>Code Snippet for a Class</Description>
+      <Shortcut>class</Shortcut>
+      <SnippetTypes>
+        <SnippetType>SurroundsWithStatement</SnippetType>
+        <SnippetType>Expansion</SnippetType>
+      </SnippetTypes>
+    </Header>
+    <Snippet>
+      <Declarations>
+        <Literal>
+          <ID>MyClass</ID>
+          <Default>MyClass</Default>
+        </Literal>
+      </Declarations>
+      <Code Language="Nodejs">
+        <![CDATA[class $MyClass$ {

```

Comment:
Let's stick to ES5. Classes are an ES6 language feature that are not currently compatible with the Node editor anyways (you'll get errors if you type this into the editor.)